src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -170,6 +170,46 @@ public static float RotationSpeedToUserReadableNumber(float rawSpeed)
         return rawSpeed * 500;
     }
 
+    public static float CalculateDigestionSpeed(float enzymeCount)
+    {
+        var amount = Constants.ENGULF_COMPOUND_ABSORBING_PER_SECOND;
+        var buff = amount * Constants.ENZYME_DIGESTION_SPEED_UP_FRACTION * enzymeCount;
+
+        return amount + buff;
+    }
+
+    public static float CalculateTotalDigestionSpeed(ICollection<OrganelleTemplate> organelles)
```

Comment:
```suggestion
    public static float CalculateTotalDigestionSpeed(IEnumerable<OrganelleTemplate> organelles)
```
to conform to existing code, also one other place where this happens.

Reply:
I'm just concerned about the performance, I initially wanted to use `IEnumerable` but `ToList` will end up going to be used. These methods will be called every frame in `HandleDigestion` so that'll be expensive.

Reply:
Unless I'm blind you *just* loop the organelles once in this method, so that shouldn't be a problem here.

Reply:
Ah okay.